hail/python/hailtop/batch/docs/change_log.rst

```diff
@@ -3,6 +3,12 @@
 Change Log
 ==========
 
+**Version 0.2.106**
+
+- Added Job.always_copy_output when using the ServiceBackend. The default behavior is False which
+is a breaking change from the previous behavior to always copy output files irregardless of the job's
```

Comment:
irregardless => regardless

Reply:
sorry. blindly copied from the original PR